ADOPTERS.md

```diff
@@ -8,6 +8,7 @@
 * [Lightbend](https://www.lightbend.com/)
 * [Red Hat](https://www.redhat.com/en)
 * [SBB CFF FFS](https://www.sbb.ch/en/home.html)
+* [Marlow Navigation](https://marlow-navigation.com/)
```

Comment:
Maybe it's not clear and @scholzj will kill me for that but can you put the name right after "Lightbend" in alphabetical order? Thanks!